op-scim-bridge/chart/op-scim-bridge-mp/values.yaml

```diff
@@ -9,3 +9,9 @@
 # otherwise the chart name "op-scim-bridge-mp" is used instead and the end result is "Referenced object not found"
 # is shown in the GCP console instead of the external IP of the service
 nameOverride: op-scim-bridge
+scim:
+  credentials:
+    volume:
+      enabled: true
+    secrets:
+      enabled: false
```

Comment:
Perhaps we should explicitly set the storage class here [like I have done in the DO version](https://github.com/digitalocean/marketplace-kubernetes/pull/469/files#diff-756df5eccfd3b933576044fd4dff3a736cba201cc9369d5772c6a0b071f75fbdR5). I believe the correct value for GCP is `"-"`